develop/reference_workarounds.rst

```diff
@@ -35,32 +35,21 @@ Citus supports all SQL statements in the multi-tenant use-case. Even in the real
 JOIN a local and a distributed table
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Attempting to execute a JOIN between a local table "local" and a distributed table "dist" causes an error:
+This error no longer occurs in the current version of Citus unless `citus.local_table_join_policy` GUC is set to `never`.
+By default, Citus follows `auto` policy. See possible options in definition of GUC.
+
+When such joins are disabled (by explicitly setting that related GUC to `never`), attempting to execute a JOIN between a local table "local" and a distributed table "dist" causes an error:
 
 .. code-block:: sql
 
   SELECT * FROM local JOIN dist USING (id);
 
   /*
-  ERROR:  relation local is not distributed
+  ERROR:  direct joins between distributed and local tables are not supported
   STATEMENT:  SELECT * FROM local JOIN dist USING (id);
   */
 
-Although you can't join such tables directly, by wrapping the local table in a subquery or CTE you can make Citus' recursive query planner copy the local table data to worker nodes. By colocating the data this allows the query to proceed.
-
-.. code-block:: sql
-
-  -- either
-
-  SELECT *
-    FROM (SELECT * FROM local) AS x
-    JOIN dist USING (id);
-
-  -- or
-
-  WITH x AS (SELECT * FROM local)
-  SELECT * FROM x
-  JOIN dist USING (id);
```

Comment:
Just removed that subquery hint since setting the guc back to `auto` should already be sufficient, any ideas ?